### PR TITLE
[issue #98]modify config name from credentials.enable to credentials.enabled

### PR DESCRIPTION
**Pull request by GuoYL123**

### CODE REVIEW

**Comment by GuoYL123** [comment]
WIP

**Comment by GuoYL123** [comment]
DO NOT MERGE

File: spring-cloud-huawei-common/src/main/java/com/huaweicloud/common/transport/ServiceCombSSLProperties.java [hunk 1]
```diff
@@ -69,11 +69,11 @@ public void setProject(String project) {
...
 
-  public boolean isEnable() {
-    return enable;
+  public boolean isEnabled() {
+    return enabled;
   }
 
-  public void setEnable(boolean enable) {
-    this.enable = enable;
+  public void setEnabled(boolean enabled) {
```
Comment (liubao68) [inline spring-cloud-huawei-common/src/main/java/com/huaweicloud/common/transport/ServiceCombSSLProperties.java]: This is a ```CHANGE```，you can do the following
1. read both old and new configuration, and print an warning messesage and suggest users use the new one
2. check if users use the old configuration, and throw and exception and force user use the new one

Comment (GuoYL123) [reply spring-cloud-huawei-common/src/main/java/com/huaweicloud/common/transport/ServiceCombSSLProperties.java]: choose the plan2

**Pull request merged**